app/screens/entry/entry.js

```diff
@@ -0,0 +1,272 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {
+    View,
+    AppState,
+    Platform,
+} from 'react-native';
+
+import DeviceInfo from 'react-native-device-info';
+import {Navigation, NativeEventsReceiver} from 'react-native-navigation';
+
+import {Client4} from 'mattermost-redux/client';
+import EventEmitter from 'mattermost-redux/utils/event_emitter';
+
+import {
+    app,
+    store,
+} from 'app/app';
+import {ViewTypes} from 'app/constants';
+import PushNotifications from 'app/push_notifications';
+import {stripTrailingSlashes} from 'app/utils/url';
+import {makeStyleSheetFromTheme} from 'app/utils/theme';
+
+
+import EmptyToolbar from 'app/components/start/empty_toolbar';
+import Loading from 'app/components/loading';
+
+const lazyLoadSelectServer = () => {
```

Comment:
these are not needed at all when you use RN's inlineRequires...

Reply:
What exactly is not needed? it's using inline requires just as stated in RN docs